Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM balena/open-balena-base:v13.4.0 as base
```

Comment:
@robbe-haesendonck I don't believe this and most of the below changes are intentional?

Reply:
Yeah, I don't exactly know what went wrong. Be it in my mind.
I'll check/rollback what's necessary when I've got the time.